### ISSUE TITLE
ansibleconfigserver_bootstrap.sh fails consistently

### ISSUE THREAD

**Issue by danni-m** [aws-quickstart/quickstart-redhat-openshift#172]
While trying to bootstrap the cluster we keep getting this error:
```
Aug 30 09:19:29 ip-10-0-137-122 cloud-init: + yum -y update
Aug 30 09:19:29 ip-10-0-137-122 cloud-init: Loaded plugins: amazon-id, rhui-lb, search-disabled-repos, versionlock
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: Resolving Dependencies
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: --> Running transaction check
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: ---> Package python-urllib3.noarch 0:1.10.2-3.el7 will be obsoleted
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: ---> Package python2-urllib3.noarch 0:1.21.1-1.el7 will be obsoleting
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: --> Finished Dependency Resolution
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: Dependencies Resolved
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: ================================================================================
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: Package           Arch     Version          Repository                    Size
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: ================================================================================
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: Installing:
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: python2-urllib3   noarch   1.21.1-1.el7     rhel-7-server-ose-3.9-rpms   173 k
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: replacing  python-urllib3.noarch 1.10.2-3.el7
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: Transaction Summary
Aug 30 09:19:31 ip-10-0-137-122 cloud-init: ================================================================================
Aug 30 09:19:32 ip-10-0-137-122 cloud-init: Install  1 Package
Aug 30 09:19:32 ip-10-0-137-122 cloud-init: Total download size: 173 k
Aug 30 09:19:32 ip-10-0-137-122 cloud-init: Downloading packages:
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: Running transaction check
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: Running transaction test
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: Transaction test succeeded
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: Running transaction
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: Installing : python2-urllib3-1.21.1-1.el7.noarch                          1/2Error unpacking rpm package python2-urllib3-1.21.1-1.el7.noarch
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: error: unpacking of archive failed on file /usr/lib/python2.7/site-packages/urllib3/packages/ssl_match_hostname: cpio: rename
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: python-urllib3-1.10.2-3.el7.noarch was supposed to be removed but is not!
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: Verifying  : python-urllib3-1.10.2-3.el7.noarch                           1/2
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: Verifying  : python2-urllib3-1.21.1-1.el7.noarch                          2/2
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: Failed:
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: python-urllib3.noarch 0:1.10.2-3.el7   python2-urllib3.noarch 0:1.21.1-1.el7
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: Complete!
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: + ((  __current_try == __tries  ))
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: + echo 'Tried 10 times and failed!'
Aug 30 09:19:33 ip-10-0-137-122 cloud-init: Tried 10 times and failed!
```

I managed to track it down to https://github.com/aws-quickstart/quickstart-redhat-openshift/blob/4418d7c0a318b6be7db4f40d3a62bd82fe036edf/scripts/ansibleconfigserver_bootstrap.sh#L52 .
Also I found this: https://bugzilla.redhat.com/show_bug.cgi?id=1187057 , they suggested to delete the urllib from pip, i manually tryed it and it worked. im not sure whats the correct fix here since there could be a version mismatch.

**Comment by pierreozoux** [comment]
@danni-m the fix is already in develop branch. It should land in master as soon as the ci build passes.

**Comment by danni-m** [comment]
@pierreozoux interesting this start to happen only in the last few days, maybe a week or two.
if this is a dup we can close this otherwise I'll close this once its fixed.

**Comment by jaymccon** [comment]
fixed by 280cc6ca627b388dabd39b67717e3515eea4627d

**Comment by danni-m** [comment]
@jaymccon @pierreozoux Thanks!
